packages/web-config-server/src/app.js

```diff
@@ -62,6 +60,7 @@ export async function createApp() {
 
   // API router
   app.use('/api/v1', getRoutesForApiV1());
+  app.use('/api/v1', logApiRequest(modelRegistry, 'tupaia', 1));
```

Comment:
Strictly speaking I suppose this should be called `web-config` rather than `tupaia`. However, I think that in an ideal world this server would be called `tupaia-server`, and I wouldn't mind kickstarting that transition. Also, since non-devs may be viewing these logs, having a user friendly name seems apt. Happy to hear reviewer's thoughts though 👍